srcpkgs/odrive/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'odrive'
+pkgname=odrive
+version=0.2.0
+revision=1
+only_for_archs="x86_64"
+makedepends=nodejs
+depends="GConf"
+short_desc="Google Drive GUI for linux"
+maintainer="Jaredy <jaredycalves@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/liberodark/ODrive"
+distfiles="https://github.com/liberodark/ODrive/archive/${version}.tar.gz"
+checksum=5e239502937711d6609eacccdcc332a2f0c8d92412d7461439ce00bf180dfaf2
+nopie=yes
+
+do_extract() {
+	tar -xvf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${version}.tar.gz --strip 1
+}
+
+do_build() {
+	cd ${wrksrc}
+	npm install
+	npm run release-linux dir
+}
+
+do_install() {
+	vmkdir ${destdir}/opt/ODrive
```

Comment:
doesn't exist, you're looking for DESTDIR, which is added implicty on relative paths.

opt/ODrive